src/normalize/entities.ts

```diff
@@ -441,11 +445,43 @@ export const Appointment = new Entity({
 	inst: ['instances', Instance]
 });
 
+export const Document = new Entity({
+	id: 'id',
```

Comment:
if key and value are same then not need to add here

Reply:
same for all below key/value pairs

Reply:
Done

Reply:
Done